lib/youtube-dl.js

```diff
@@ -305,3 +305,22 @@ ytdl.getSubs = function(url, options, callback) {
     callback(null, files);
   });
 };
+
+/**
+ * @param {Boolean} descriptions
+ * @param {Object} options
```

Comment:
forgot to mention to include a `!` to denote that these two arguments (`descriptions` and `options`) are optional

like [this](https://github.com/fent/node-youtube-dl/blob/master/lib/youtube-dl.js#L33)